.devcontainer/devcontainer.json

```diff
@@ -1,20 +1,31 @@
 {
     "name": "CHIP Ubuntu Development Environment",
     "runArgs": [
-        "--cap-add=SYS_PTRACE",
-        "--security-opt",
-        "seccomp=unconfined",
         "--network=host",
-        "--privileged",
-        "-v",
-        "/dev/bus/usb:/dev/bus/usb:ro",
         "--device-cgroup-rule=a 189:* rmw",
         "--add-host=host.docker.internal:host-gateway"
     ],
+    "privileged": true,
+    "capAdd": ["SYS_PTRACE"],
+    "securityOpt": ["seccomp=unconfined"],
     "mounts": [
-        "source=/var/run/docker.sock,target=/var/run/docker.sock,type=bind"
+        {
+            "source": "/var/run/docker.sock",
+            "target": "/var/run/docker.sock",
+            "type": "bind"
+        },
+        {
+            "source": "/dev/bus/usb",
+            "target": "/dev/bus/usb",
+            "type": "bind"
+        },
+        {
+            "source": "/run/dbus/system_bus_socket",
```

Comment:
Will it work on Mac OS (does directory like that exists on Mac)?

Reply:
I reverted last commit, the container won't start if one of the mounts does not exist.